src/loader-components/_ExampleTemplate.jsx

```diff
@@ -58,26 +92,34 @@ const ExampleTemplate = ({
         {example}
       </div>
       {exampleSrc
-        && (
-        <div className={cx('footer')}>
-          <div className={cx('button-container')}>
-            <button type="button" className={cx('bg-toggle')} onClick={() => setIsBackgroundTransparent(!isBackgroundTransparent)}>
-              Toggle Background
-            </button>
-            <button type="button" className={cx('code-toggle')} onClick={() => setCodeIsVisible(!codeIsVisible)}>
-              <IconChevronLeft className={cx('chevron')} />
-              <span>Code</span>
-              <IconChevronRight className={cx('chevron')} />
-            </button>
-          </div>
-          {codeIsVisible
-            && (
-            <div className={cx('code')}>
-              {exampleSrc}
+          && (
+          <div className={cx('footer')}>
+            <div className={cx('button-container')}>
+              {cssButtonVisible && (
+                <button type="button" className={cx('css-toggle', { 'is-selected': isCssSelected })} onClick={handleCssToggle}>
+                  CSS
+                </button>
+              )}
+              <button type="button" className={cx('code-toggle', { 'is-selected': isCodeSelected })} onClick={handleCodeToggle}>
+                <IconChevronLeft className={cx('chevron')} />
+                <span>Code</span>
+                <IconChevronRight className={cx('chevron')} />
+              </button>
             </div>
-            )}
-        </div>
-        )}
+            {cssIsVisible
+              && (
+                <div className={cx('css')}>
+                  {exampleCssSrc}
+                </div>
+              )}
+              {codeIsVisible
+              && (
+                <div className={cx('code')}>
+                  {exampleSrc}
+                </div>
+              )}
+          </div>
+          )}
```

Comment:
This is pretty big, can we break this logic out of the return?

Reply:
Broke out this logic into a `renderExamples` constant and did the same with the buttons into a `renderButtons`: [2e28aa6](https://github.com/cerner/terra-dev-site/pull/281/commits/2e28aa6167c3cf422dbd74674a56591b9ecbdf9b).